build.cake

```diff
@@ -8,6 +8,7 @@ using Semver;
 
 var target = Argument<string>("target", "Default");
 var configuration = Argument<string>("configuration", "Debug");
+var buildFramework = configuration.EndsWith("_NetCore") ? "netcoreapp2.1" : "net45";
```

Comment:
You are testing here for `EndsWith("_NetCore")`, in the CKAN-Core.csproj you are testing for `EndsWith("NetCore")`.
It's not critical, but it would be nice if it's consistent.